src/app/src/components/DrawMap/DrawMap.js

```diff
@@ -8,11 +8,13 @@ import Map from '../Map';
 import EditToolbar from './EditToolbar';
 import MapControlButtons from './MapControlButtons';
 import MapFeatures from './MapFeatures';
+import MunicipalBoundariesLayer from './MunicipalBoundariesLayer';
```

Comment:
Consider moving this file out of `DrawMap` and into `components/Layers/MunicipalBoundaries.js`. We may also be able to move `Basemaps.js` in there.